desktop/cmp/input/JsonInput.ts

```diff
@@ -4,15 +4,20 @@
  *
  * Copyright © 2025 Extremely Heavy Industries Inc.
  */
-import {hoistCmp} from '@xh/hoist/core';
+import {hoistCmp, PlainObject} from '@xh/hoist/core';
 import '@xh/hoist/desktop/register';
 import {fmtJson} from '@xh/hoist/format';
-import * as codemirror from 'codemirror';
-import 'codemirror/mode/javascript/javascript';
+import Ajv, {Options, ValidateFunction} from 'ajv';
 import {codeInput, CodeInputProps} from './CodeInput';
-import {jsonlint} from './impl/jsonlint';
 
-export type JsonInputProps = CodeInputProps;
+export interface JsonInputProps extends CodeInputProps {
+    jsonSchema?: PlainObject;
```

Comment:
I did some testing in toolbox where I defined the schema and saw it working. It was not too hard, but it might be nice to add a comment here with some basics about the shape of the validation schema, and/or links to the AJV page showing how to get started with writing the validation.

Actually typing it seems like a nightmare, but we should provide some initial steps to get people started using it.